src/main/java/core/basesyntax/service/Report.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface Report {

```

Comment:
I think ReportMaker (for example) is better name for this interface than just Report